### ISSUE TITLE
rnc-cli ENOENT error - RNC_CLI_BINARY_PATH has incorrect path in monorepos

### ISSUE THREAD

**Issue by marklawlor** [callstack/react-native-builder-bob#691]
### Description

https://github.com/callstack/react-native-builder-bob/pull/685 changed how the `@react-native-community/cli` was invoked and now it incorrectly assumes that the binary will be located within `process.cwd()/node_modules/bin`. In a monorepo structure, it is unlikely the binary is in this location due to package hoisting.

Previously, bob used `npx` to invoke the binary (which respects monorepo structure), but after the PR the path to the binary is hardcoded as

```
  const RNC_CLI_BINARY_PATH = path.resolve(
    process.cwd(), // We are always expected to run in the library
    'node_modules',
    '.bin',
    rncCliBinaryName
  );
```

### Packages

- [ ] create-react-native-library
- [x] react-native-builder-bob

### Selected options

-

### Link to repro

-

### Environment

-

**Comment by atlj** [comment]
Thanks for this, this is the second issue about #685 breaking workflows. I'm going to revert back as the original intention was to be able to call bob from xcode.

**Comment by atlj** [comment]
@marklawlor we just published [bob@0.33.1](https://github.com/callstack/react-native-builder-bob/releases/tag/react-native-builder-bob%400.33.1) which includes the fix.
